api/v1alpha3/types.go

```diff
@@ -133,6 +133,10 @@ type VirtualMachineCloneSpec struct {
 	// Defaults to empty map
 	// +optional
 	CustomVMXKeys map[string]string `json:"customVMXKeys,omitempty"`
+
+	// Tags is an optional set of tags to add to an instance.
+	// +optional
+	Tags []string `json:"tags,omitempty"`
```

Comment:
Should we highlight that this is tagID or do we plan to support tag resolution and caching?